docs/php/how-to-develop-an-activity-definition-in-php.md

```diff
@@ -0,0 +1,54 @@
+---
+id: how-to-develop-an-activity-definition-in-php
+title: Develop an Activity Definition in PHP
+sidebar_label: Activity Definition
+---
+
+Activities are defined as methods of a plain PHP interface annotated with `#[ActivityInterface]`.
+(You can also use PHP 8 attributes in PHP 7.)
+
+Each method defines a single Activity type.
+A single Workflow can use more than one Activity interface and call more than one Activity method from the same interface.
+
+The only requirement is that Activity method arguments and return values are serializable to a byte array using the provided [DataConverter](https://github.com/temporalio/sdk-php/blob/master/src/DataConverter/DataConverterInterface.php) interface.
+The default implementation uses a JSON serializer, but an alternative implementation can be easily configured.
+
+Following is an example of an interface that defines four Activities:
+
+```php
+#[ActivityInterface]
+interface FileProcessingActivities
+{
+    public function upload(string $bucketName, string $localName, string $targetName): void;
+
+    #[ActivityMethod("transcode_file")]
+    public function download(string $bucketName, string $remoteName): void;
+
+    public function processFile(): string;
+
+    public function deleteLocalFile(string $fileName): void;
+}
+```
+
+We recommend to use a single value type argument for Activity methods.
+In this way, adding new arguments as fields to the value type is a backward-compatible change.
+
+An optional `#[ActivityMethod]` annotation can be used to override a default Activity name.
```

Comment:
Let's have an h4 section that shows how to specify the Activity Type/Name - "how to customize activity type"